docs/src/componentDocs/CollapsibleHeaderLayout/playground/PlaygroundPage.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useRef } from 'react';
+import Box from '@mui/material/Box';
+import {
+    InputConfig,
+    PreviewComponent,
+    CodeSnippetFunction,
+    getPropsToString,
+    getPropsMapping,
+    Playground,
+} from '@brightlayer-ui/react-doc-components';
+import { CollapsibleHeaderLayout, HeaderProps } from '@brightlayer-ui/react-native-components';
+import { getBodyFiller, DRAWER_WIDTH } from '../../../utils';
+import 'prismjs/components/prism-jsx.min';
+import { View } from 'react-native';
+
+const inputConfig: InputConfig = [
+    // Required Props
+    // Optional Header Props
+    {
+        id: 'collapsedHeight',
+        label: 'collapsedHeight',
+        type: 'number',
+        description: 'The height of the header when collapsed',
+        required: false,
+        initialValue: 56,
+        minValue: 56,
+        maxValue: 180,
+        defaultValue: 56,
+        category: 'Optional Props',
```

Comment:
These are not optional props, they are Header props...look at how we did this for the Hero (ChannelValue Props). You just need to change the name of the category to 'Header props' for consistency. (These are not optional props on the CollapsibleHeaderLayout...they are not props on the CollapsibleHeaderLayout at all. The prop on the CollapsibleHeaderLayout is called `HeaderProps`...which is where these values will get passed into).